python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -1,27 +1,35 @@
 #!/usr/bin/env python
 
 import matplotlib
-matplotlib.use("Agg")
+matplotlib.use("Agg")  # noqa #402
 import matplotlib.pyplot as plt
 import numpy as np
-np.seterr(all="ignore")
+np.seterr(all="ignore")  # noqa #402
 import functools
+import os
+import scipy.stats as scipyStats
 
 from collections import defaultdict
 
 from lsst.pex.config import Config, Field, ConfigField, ListField, DictField, ConfigDictField
-from lsst.pipe.base import CmdLineTask, ArgumentParser, TaskRunner
+from lsst.pipe.base import CmdLineTask, ArgumentParser, TaskRunner, TaskError
 from lsst.coadd.utils import TractDataIdContainer
 from .analysis import Analysis, AnalysisConfig
 from .coaddAnalysis import CoaddAnalysisTask
-from .utils import *
-from .plotUtils import *
-
+from .utils import (Filenamer, Enforcer, concatenateCatalogs, checkIdLists, getFluxKeys, addPatchColumn,
+                    calibrateCoaddSourceCatalog, fluxToPlotString, writeParquet, getRepoInfo,
+                    orthogonalRegression, distanceSquaredToPoly, p2p1CoeffsFromLinearFit, makeEqnStr,
+                    catColors)
+from .plotUtils import OverlapsStarGalaxyLabeller, labelCamera, setPtSize
+
+import lsst.afw.coord as afwCoord
```

Comment:
Thanks for moving this up here. Turns out there's no use of afwCoord in this file anymore.